larky/src/main/resources/vendor/luhn.star

```diff
@@ -0,0 +1,44 @@
+load("@stdlib//larky", "larky")
+
+def _sum(nums):
+    s = 0
+    for i in nums:
+      s += i
+    return s
+   
+def _mod_digits(d):
+    if d < 10:
+        return d
+    else:
+        return ((d % 10) + (d // 10))
+
+def _luhn_summation(nums,index):
+    nums = nums[index::-1]
+    doubledArr = []
+    for i in range(len(nums)):
+      if(i%2==0):
+        doubledArr.append(_mod_digits(int(nums[i]) * 2))
+      else:
+        doubledArr.append(int(nums[i]))
+    luhn_sum = _sum(doubledArr)
+    return str(luhn_sum)
+
+def verify(num):
+    checksum = int(num[-1])
+    cardSum = int(_luhn_summation(num,-2))
+    return ((cardSum + checksum) % 10) == 0
+
+def generate(num):
+    cardSum = int(_luhn_summation(num,-1))
+    modulated = cardSum % 10
+    return 10 - modulated
+
+def append(num):
+    return num + str(generate(num))
+
```

Comment:
can we copy the docstrings as well in the function from here? 

https://github.com/mmcloughlin/luhn/blob/master/luhn.py#L3-L43

Reply:
Added them:
https://github.com/OmegaPointZero/starlarky/blob/bc58a870acf4683607ee81934ee0ab7538a8ad8e/larky/src/main/resources/vendor/luhn.star#L40-L46